src_erl/Communication_Layer/http_Nerlserver/src/source/parser.erl

```diff
@@ -23,12 +23,12 @@
 %%this parser takes a CSV folder containing chunked data, parsing into a list of binary.
 %%each record in the line is a batch of samples
 parse(ChunkSize,FolderName)->
-%%  FolderName="./../input/shuffled-input1_splitted/",
+%%  FolderName="./input/shuffled-input1_splitted/",
```

Comment:
change this module name to csvParser (since we might have more parsers in the future). 
@kapelnik